src/icons/general/IconGlobe2.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconGlobe2 = ({ className = '', color = '#8EA6B2', height = 12, title, width = 12 }: Props) => (
```

Comment:
color should come from the variables

Reply:
This color doesn't exist in variables but is used in a few other icons. Can I just add it?